test/TestUseHandlerRatherThanWhenChanged.py

```diff
@@ -1,10 +1,10 @@
 import unittest
+from test import RunFromText
```

Comment:
isort should be made aware of the first-party test helpers. It's best to use an explicit relative import for this.